server/chat-plugins/scavengers.js

```diff
@@ -416,6 +489,26 @@ class ScavengerHunt extends Rooms.RoomGame {
 			reset = true;
 		}
 
+		if (!ScavengerHuntDatabase.isEmpty() && this.room.addRecycledHuntsToQueueAutomatically) {
+			if (!this.room.scavQueue) {
+				this.room.scavQueue = [];
+			}
+
+			const next = ScavengerHuntDatabase.getRecycledHuntFromDatabase();
+			const correctlyFormattedQuestions = next.questions.reduce((alreadyFormatted, questionObject) => {
```

Comment:
Please use `flatMap` instead. Using `reduce` like this is very confusing, and unnecessary.